app/views/user_mailer/invite_from_community_email.text.erb

```diff
@@ -1,7 +1,7 @@
-# You've been invited to join the Buckinghamshire Family Information Service
+# You've been invited to join <%= ENV['OUTPOST_INSTANCE_NAME'].presence || 'Outpost' %>
```

Comment:
@em-cd  Outpost for admin is a joint name but outposts name for users is different

the Buckinghamshire Family Information Service
the Buckinghamshire Online directory